tests/forms_tests/tests/test_fields.py

```diff
@@ -947,28 +947,28 @@ def test_booleanfield_1(self):
         f = BooleanField()
         self.assertRaisesMessage(ValidationError, "'This field is required.'", f.clean, '')
         self.assertRaisesMessage(ValidationError, "'This field is required.'", f.clean, None)
-        self.assertEqual(True, f.clean(True))
+        self.assertTrue(f.clean(True))
```

Comment:
I don't think we should change this since `self.assertTrue()` will pass for any "truthy" value.